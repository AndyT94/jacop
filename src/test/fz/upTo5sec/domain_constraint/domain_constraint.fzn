var 5..5: val:: output_var;
array [1..4] of var int: var01:: output_array([1..4]) = [0,1,0,0];
array [1..4] of var int: values:: output_array([1..4]) = [9,5,2,7];
solve  satisfy;
