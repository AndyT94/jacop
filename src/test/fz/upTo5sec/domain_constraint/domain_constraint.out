val = 5;
var01 = array1d(1..4, [0, 1, 0, 0]);
values = array1d(1..4, [9, 5, 2, 7]);
----------